src/registration/scripts/directives.js

```diff
@@ -13,6 +14,17 @@
                 link: function (scope, element, attrs) {
                     scope.data = {};
 
+                    // Bind on-success attribute to success callback
+                    // scope: { onRegister: '&' }
+                    if (angular.isDefined(attrs.onRegister)) {
```

Comment:
I was trying to avoid non-breaking changes and I thought that isolating the scope may be `breaking`.